causes_service/causes/search.py

```diff
@@ -84,4 +86,13 @@ def delete_search_index(self, client: meilisearch.Client):
         serializer=NewsArticleSerializer,
         queryset=NewsArticle.objects.all()
     ),
-]
+    ModelSearchIndex(
+        index_name='blogs',
+        searchable_attributes=['title','authors.name','body'],
+        filterable_attributes=['authors.id'],
+        sortable_attributes=['release_at_timestamp'],
+        model=Blog,
+        serializer=BlogSerializer,
+        queryset=Blog.objects.filter_active(is_active_at=timezone.now())
```

Comment:
It looks we are already filtering by active in the populate active method, so we can just have Blog.objects.all() here.

https://github.com/now-u/now-u-backend/blob/6b4b1f2389b030252cb65ae441d576369122fb04/causes_service/causes/search.py#L28

Reply:
Updated.